tests/e2e/core-tests/specs/merchant/wp-admin-settings-shipping-classes.test.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jest/no-export*/
+
+/**
+ * Internal dependencies
+ */
+const { merchant } = require('@woocommerce/e2e-utils');
+const { lorem, helpers } = require("faker");
+
+const runAddShippingClassesTest = () => {
+	describe('Merchant can add shipping classes', () => {
+		beforeAll(async () => {
+			await merchant.login();
+
+			// Go to Shipping Classes page
+			await merchant.openSettings('shipping', 'classes');
+		});
+
+		it('can add shipping classes', async () => {
+			const shippingClassSlug = {
+				name: lorem.words(),
+				slug: lorem.slug(),
+				description: lorem.sentence()
+			}
```

Comment:
We haven't set up a linter yet. But, when implemented it reject script that relies on automatic semi-colons (similar to [WordPress standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#semicolons). Can you add the semi-colons?

Reply:
Thanks @rrennick
I have added the semi-colons as well as fixed other styling issues. Thank you for pointing me to the WordPress coding standards.
Fixed in https://github.com/woocommerce/woocommerce/pull/29467/commits/e39badbcf71494870e06619a3c37b6e883a80fcb